exercises/dominoes/domino.rb

```diff
@@ -0,0 +1,16 @@
+class Domino
+  attr_reader :left, :right
+
+  def self.deep_copy(domino)
+    Domino.new(domino.left, domino.right)
+  end
```

Comment:
Yes, I believe I can use clone method. Need to investigate differences between using clone and dup in this context when I have a minute.

"Serena Sees Her Footprints on the Moon" is on sale now @ iTunes! 
https://itun.es/us/CnX6bb.l

It's the story of a girl who imagines a trip to the Moon, where she interacts with some of the objects left behind up there by the astronauts from the Apollo missions.

No iPad or iPhone? No problem!
Get the book here: https://gum.co/softcover

> On Sep 1, 2016, at 1:55 PM, Tute Costa <notifications@github.com> wrote:
> 
> In exercises/dominoes/domino.rb:
> 
> > @@ -0,0 +1,16 @@
> > +class Domino
> > +  attr_reader :left, :right
> > +
> > +  def self.deep_copy(domino)
> > +    Domino.new(domino.left, domino.right)
> > +  end
> Is this the same as https://ruby-doc.org/core-2.3.1/Object.html#method-i-clone? If so, can we use the Ruby method?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or mute the thread.
>